rainier-core/src/main/scala/com/stripe/rainier/core/Injection.scala

```diff
@@ -17,21 +17,29 @@ trait Injection { self =>
     See https://en.wikipedia.org/wiki/Probability_density_function#Dependent_variables_and_change_of_variables
     This function should be log(d/dy backwards(y)), where y = forwards(x).
    */
-  def logJacobian(y: Real): Real
+  def logBackwardsJacobian(y: Real): Real
+  def logForwardsJacobian(x: Real): Real
 
   def transform(dist: Continuous): Continuous = new Continuous {
     def realLogDensity(real: Real): Real =
       If(isDefinedAt(real),
```

Comment:
random note I didn't have anywhere else to include: if we add a `Support` trait we should move `isDefinedAt` to there

Reply:
Done